### PR TITLE
Disable fault injection for TestGetProperty

### PR DESCRIPTION
**Pull request by hx235**
**Context/Summary:** 
See titled; along with one more minor fix to other disabling

**Test:** CI won't show `Failed to get DB property: rocksdb.aggregated-table-properties`

### CODE REVIEW

File: db_stress_tool/db_stress_test_base.cc [hunk 1]
```diff
@@ -1230,26 +1255,30 @@ void StressTest::OperateDb(ThreadState* thread) {
...
           // backup/restore after fixing the various issues it surfaces
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kMetadataRead);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kMetadataWrite);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kRead);
-          fault_fs_guard->DisableThreadLocalErrorInjection(
-              FaultInjectionIOType::kWrite);
+          if (fault_fs_guard) {
```
Comment (jaykorean) [inline db_stress_tool/db_stress_test_base.cc:1261]: Just for a thought (Not about this PR) 

I see we do this in a few different places. Just wondering if it's worth creating a macro for this for convenience. (e.g. if we add a new FaultInjectionIOType, we only have to add this to a single place)

**Review by jaykorean — APPROVED** [review]
Thanks!

File: db_stress_tool/db_stress_test_base.cc (hunk 1 above)
Comment (hx235) [reply db_stress_tool/db_stress_test_base.cc:1261]: yeah could be a good follow up.

**Comment by pdillinger** [comment]
I believe the crash test failure revealed an API problem: @hx235 

>   // If "property" is a valid "string" property understood by this DB
>  // implementation (see Properties struct above for valid options), fills
>  // "*value" with its current value and returns true.  Otherwise, returns
>  // false.

It doesn't say the function can return false because of IO failures. So IMHO this PR is just suppressing a failure, not a "fix" to the underlying problem.

**Comment by hx235** [comment]
> I believe the crash test failure revealed an API problem: @hx235
> 
> > // If "property" is a valid "string" property understood by this DB
> > // implementation (see Properties struct above for valid options), fills
> > // "*value" with its current value and returns true.  Otherwise, returns
> > // false.
> 
> It doesn't say the function can return false because of IO failures. So IMHO this PR is just suppressing a failure, not a fix to the underlying problem. From the PR description, it wasn't clear if that was known / intended.

I see .. let me fix the API doc too

**Pull request closed without merging**